style.css

```diff
@@ -94,9 +94,24 @@ ul { list-style-type: square; }
 
 .episode {
 	border-top: 1px dashed #d6611b;
-	margin-top: 2em;
-	padding-top: 2em;
+	margin-top: 1em;
+	padding-top: 1em;
 }
+	.episode_decor {
+	  float: left;
+		padding-top: 0.25em;
```

Comment:
There seem to be some weird indentation going on here?

Reply:
Will fix.